transmission/src/main/java/com/trendyol/transmission/router/RequestDelegate.kt

```diff
@@ -71,13 +71,17 @@ internal class RequestDelegate(
         query: Query.Computation
     ) = queryScope.launch {
         val computationHolder = routerRef.transformerSet
-            .find { it.storage.hasComputation(query.key) }
+            .find { it.storage.hasComputation(query.key) } ?: return@launch
```

Comment:
This will cause us to not return a computation result to the client. If we can't find a computationHolder for some reason, result should be null

Reply:
IMO, If the computationHolder(aka Transformer) is null, we shouldn't return a computation result to the client anyway. That's why I've added an early return to it.

Reply:
Got it, thank you. Updated 🙏